### DISCUSSION TITLE
no puedo instalar el compilador

### DISCUSSION THREAD

**Discussion by lucialemusss** [INGCOM-UNRN-P1/cursada-2023#19] [Preguntas y respuestas] [answered]
Buenas.. intente seguir la guía pero cuando llego al momento en el que hay que editar la variable de entorno ya no me aparece igual... mi compu tiene windows 7 creo que ese es el problema, en este caso no sabría como avanzar.

**Comment by martinvilu** [comment]
Lo que tenés que hacer es agregar a mano el texto en la variable de entorno, tené cuidado de no borrar lo que estaba antes.

Mandá capturas de eso y te guio puntualmente ahí.

También, antes de que avances con eso, también compartí qué computadora es, ya que con Windows 7, tal vez necesites otras versiones del software para que todo funcione.

**Reply by lucialemusss** [reply]
Es una samsung modelo NC210/NC110 

perdón nose si eso responde la pregunta(? jaja

**Reply by martinvilu** [reply]
Si, el modelo de computadora a simple vista no parece necesitar de software especial.

Faltaría que envíes las capturas de pantalla.

**Reply by lucialemusss** [reply]
![cap](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141846852/b6080177-8cfc-4b3a-a147-39348d50bae8)

**Comment by martinvilu** [comment]
Cuando le das `Editar...`, agregá al final un `;` y la dirección como esta en el wiki `C:\mingw64\bin` (o donde esté descompactado`; si no coincide, vas a tener el mismo error que antes; que "no es un programa reconocido"

![1o69d](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/2dd4272e-3e2a-47fc-b26f-f8f462e37c2f)

**Reply by lucialemusss** [reply]
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141846852/20d30bd1-5c4a-4585-b89b-a74fc9622629)

**Reply by lucialemusss** [reply]
no funciono AAAAA

**Reply by martinvilu** [reply]
En donde descompactaste el archivo del mingw?

Manda lo que muestra la consola luego de (solo lo que esta en gris):
`c:` (va a la raíz del disco)
`dir` (muestra el contenido

**Comment by JeanFranco99** [comment]
Lo instalaste en el directorio `C:\mingw64`?
Lo marcaste en el instalador asi? 
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/83616156/b7ca3809-db16-4f26-988b-3b541e952d1e)
Por las dudas, desinstala y volve a instalar siguiendo de  esta [pagina](https://platzi.com/tutoriales/1189-algoritmos-2017/1901-como-instalar-gcc-para-compilar-programas-en-c-desde-la-consola-en-windows/)

**Reply by martinvilu** [reply]
La ubicación en donde está el archivo es crítico, sin eso lo otro no funcionara

**Reply by lucialemusss** [reply]
Lo bueno es que ese problema esta descartado..😅 
Lo malo es que hay otro.. :'v

**Reply by lucialemusss** [reply]
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141846852/ae16a742-3ced-4473-9846-32d5afc7e421)

**Reply by martinvilu** [reply, marked as answer]
Funciona! pero no completamente;

Vas a tener que descargar este: [gcc-i686](https://github.com/brechtsanders/winlibs_mingw/releases/download/13.2.0-16.0.6-11.0.0-ucrt-r1/winlibs-i686-posix-dwarf-gcc-13.2.0-llvm-16.0.6-mingw-w64ucrt-11.0.0-r1.zip)

Como en lugar de `mingw64`, es `mingw32`, borrá la carpeta original, y al descompactar la nueva, cambiale el nombre que tenía la anterior.
